internal/configs/transportserver.go

```diff
@@ -111,13 +113,15 @@ func generateStreamUpstreams(transportServerEx *TransportServerEx, upstreamNamer
 	var upstreams []version2.StreamUpstream
 
 	for _, u := range transportServerEx.TransportServer.Spec.Upstreams {
+		_, isExternal := transportServerEx.ExternalNameSvcs[GenerateExternalNameSvcKey(transportServerEx.TransportServer.Namespace, u.Service)]
```

Comment:
You can move this line down to line 123. This puts the code for getting the value of `isExternal` right above the code that actually uses it. i.e. `ups.Resolve = isExternal`

This makes it easier for the reader to know why this variable is used.